docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -348,6 +348,313 @@ and a file system location of ``/var/my_tables/test_table``::
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
 
+.. _iceberg-metadata-tables:
+
+Metadata tables
+---------------
+
+The Iceberg connector maintains several hidden tables that provide metadata
+for a specific table. You can query each metadata table by appending the
+metadata table name to the table name::
+
+   SELECT * FROM "test_table$data"
+
+``$data`` table
+^^^^^^^^^^^^^^^
+
+The ``$data`` table is an alias for the Iceberg table.
+
+The statement::
+
+    SELECT * FROM "test_table$data"
+
+is equivalent to::
+
+    SELECT * FROM test_table
+
+``$properties`` table
+^^^^^^^^^^^^^^^^^^^^^
+
+The ``$properties`` table provides access to general information about iceberg
+table configuration and any additional metadata key/value pairs that the table
+was tagged with.
+
+You can retrieve the changelog of the Iceberg table ``test_table``
```

Comment:
In what way is it a `changelog`. It is current state of properties. Right?

Reply:
This is a mistake. Sorry.